api_tests/config.py

```diff
@@ -22,10 +22,11 @@ def parse():
 
     # Make sure config file has all required keys
     missing_key = verify_keys(config_dict, [
-      'url',
-      'access_token',
-      'snippet',
-      'user_id',
+        'url',
+        'access_token',
+        'snippet',
+        'user_id',
+        'invalid_snippet_key',
```

Comment:
The test can be run after the [DELETE test](https://github.com/maryvilledev/codesplain-lambdas/blob/master/api_tests/users/user_id/snippets/snippet_id/test_endpoint.py#L24) is performed. At that point, we know for sure of one invalid ID: the one that way just deleted.